.github/workflows/preview.yaml

```diff
@@ -0,0 +1,43 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Preview web app
+
+# Controls when the action will run. Triggers the workflow on push or pull request 
```

Comment:
Not on PRs cause that would be a security issue (code is right just not the comment).